### DISCUSSION TITLE
New release soon?

### DISCUSSION THREAD

**Discussion by yvovandoorn** [pretalx/pretalx#1359] [Self-hosting] [answered]
It's been 10 months since 2.3.1 dropped and y'all have certainly not stopped developing. Will there be a new release soon?

**Comment by rixx** [comment, marked as answer]
I was hoping to release pretalx 3.0 next, which would include some ~breaking build changes and a new API. Target for that would be the end of the year.

If you want to get the [latest features](https://docs.pretalx.org/en/latest/changelog.html) right now – the main branch is installable 99% of the time (it's what's powering pretalx.com), and the current commit, [11652a5](https://github.com/pretalx/pretalx/commit/11652a5857316b2280b16305621a0088d19adf12), is definitely safe. We have documentation on how to install a commit [here](https://docs.pretalx.org/administrator/maintenance.html#installing-a-fixed-release) (pip supported, thankfully).

**Comment by yvovandoorn** [comment]
That makes a ton of sense. Thanks for the background. I found a small issue when using your instructions but #1360 fixes it.
